model/task_lifecycle.go

```diff
@@ -1703,6 +1703,13 @@ func checkResetDisplayTask(t *task.Task) error {
 			Status: evergreen.TaskFailed,
 		}
 	}
+	grip.Debug(message.Fields{
```

Comment:
Would it make sense to only log if details are nil to limit logging?

Reply:
We wouldn't want to log if details are nil since we know that it will be manually set in this function if that is the case.  However I can wrap this in a conditional for when `details.Status == ""` since that is the unexpected case we're looking for.